wallet/wallet_test.go

```diff
@@ -205,3 +206,43 @@ func TestLabelTransaction(t *testing.T) {
 		})
 	}
 }
+
+// TestGetTransaction writes a tx to disk and then compares
+// the hash to the response from the GetTransaction function.
+func TestGetTransaction(t *testing.T) {
```

Comment:
The test could be more comprehensive, I think we should at least test three cases,
- when it's not found
- when it's not mined
- when it's confirmed

Reply:
Makes sense. I will add them.